app/api/aggregate_outbreaks.py

```diff
@@ -372,3 +372,38 @@ def sum_outbreaks(df):
     processed_df = df.groupby(
         ['Facility', 'County', 'State_Facility_Type'], as_index=False).apply(process_sum_outbreaks)
     return processed_df
+
+
+def nj_special_aggregator(df):
+    def process_facility(df):
+        col_map = utils.make_matching_column_name_map(df)
+        df['Outbrk_Status'] = df['Outbrk_Status'].str.upper()
```

Comment:
I think this part should be taken care of in the standardization step? (If it isn't in practice, let me know!)

Reply:
Ah yeah that bit was redundant. Removed